setup.py

```diff
@@ -21,17 +15,10 @@
     description="Runs a thing that analyzes experiments",
     url="https://github.com/mozilla/pensieve",
     packages=["pensieve", "pensieve.tests"],
-    install_requires=[
-        "attrs",
-        "cattrs",
-        "incremental",
-        "pytz",
-        "requests",
-    ],
+    install_requires=text_from_file("requirements.txt").strip().split("\n"),
```

Comment:
I thought we wanted to list abstract dependencies here and concrete pinned dependencies in requirements.txt; we'd need to maintain both separately.

Reply:
True, I was hoping to reduce a little bit of duplication here